content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: blog
+title: Prow and Tide for Kubernetes Contributors
+date: 2022-05-31
+slug: prow-and-tide-for-kubernetes-contributors
+---
+
+**Authors:** [Chris Short](https://github.com/chris-short), [Frederico Muñoz](https://github.com/fsmunoz)
```

Comment:
Fredrico,

Given the amount of effort you put into this, your name should go first, in my opinion.

Reply:
Thank you for that, but I'm happy with how it is and think it reflects well the work done - plus, it's sorted alphabetically!